source/plugins/ruby/in_kube_nodes.rb

```diff
@@ -90,6 +97,7 @@ def shutdown
           @condition.signal
         }
         @thread.join
+        super # This super must be at the end of shutdown method
```

Comment:
can u explain why super is called in the end ?

Reply:
This is upgrade guidance from fluentd. please see examples in this doc - https://docs.fluentd.org/plugin-development/plugin-update-from-v0.12